src/battle-scene.ts

```diff
@@ -2625,9 +2625,7 @@ export default class BattleScene extends SceneBase {
   getWaveMoneyAmount(moneyMultiplier: number): number {
     const waveIndex = this.currentBattle.waveIndex;
     const waveSetIndex = Math.ceil(waveIndex / 10) - 1;
```

Comment:
```suggestion
    const waveSetIndex = Math.ceil(waveIndex / 10) ;
```
`waveSetIndex` being 0 for waves 1-10 causes `moneyValue` to spike in the first 10 waves since

```
(waveIndex - 1) % 10 = -1 % 10 = 9
```